dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarLinkageReport.java

```diff
@@ -36,9 +36,11 @@
    */
   abstract ImmutableList<DependencyPath> getDependencyPaths();
 
-  abstract ImmutableList<LinkageErrorMissingClass> getMissingClassErrors();
-  abstract ImmutableList<LinkageErrorMissingMethod> getMissingMethodErrors();
-  abstract ImmutableList<LinkageErrorMissingField> getMissingFieldErrors();
+  abstract ImmutableList<LinkageErrorOnReference<ClassSymbolReference>> getMissingClassErrors();
```

Comment:
Maybe this is just LinkageError? Does "OnReference" add any information?

Reply:
No it doesn’t add any information. I like “LinkageError”.

If I remember correctly you didn’t like class named “ XXXError” when it’s not inheriting Error.

Reply:
`LinkageError` seems like a problem because it is the same name as the standard library class: https://docs.oracle.com/javase/7/docs/api/java/lang/LinkageError.html

Reply:
`StaticLinkageError`?